lib/src/event.dart

```diff
@@ -322,27 +322,28 @@ class Event extends MatrixEvent {
     return receiptsList;
   }
 
-  /// Removes this event if the status is [sending], [error] or [removed].
-  /// This event will just be removed from the database and the timelines.
-  /// Returns [false] if not removed.
+  @Deprecated('Use [delete()] instead.')
   Future<bool> remove() async {
-    final room = this.room;
-
-    if (!status.isSent) {
-      await room.client.database?.removeEvent(eventId, room.id);
-
-      room.client.onEvent.add(EventUpdate(
-        roomID: room.id,
-        type: EventUpdateType.timeline,
-        content: {
-          'event_id': eventId,
-          'status': EventStatus.removed.intValue,
-          'content': {'body': 'Removed...'}
-        },
-      ));
+    try {
+      await delete();
       return true;
+    } catch (_) {
+      return false;
     }
-    return false;
+  }
+
+  /// Removes this event from the database and a timeline.
+  /// This event will just be removed from the database and the timelines.
+  /// This is only possible for events which are not sent yet, so have a
+  /// status of `SENDING` or `ERROR`. Throws an exception if used for an already
+  /// sent event!
+  Future<void> delete() async {
```

Comment:
do you think naming this something like `cancelSend` makes it more obvious?

Reply:
Good idea 🤔

Reply:
Changed